### DISCUSSION TITLE
Deploy serverless functions without a static site

### DISCUSSION THREAD

**Discussion by tafelito** [vercel/vercel#4000] [General] [answered]
I'm trying to deploy a bunch of serverless functions to Zeit Now without having any static site, just the node.js functions. This was possible before but since latest version I'm having a few issues. When I tried to deploy for the first time, it was asking for a public directory. Since I don't have a static site, I don't have a public dir. So I went to the project settings and set my `dist` folder as output. That allowed me to do the deploy. But now if I try to run now dev, it will throw the same error `No Output Directory named "public" found after the Build completed. You can configure the Output Directory in your project settings.` 

How can I fix this?

**Comment by paulogdm** [comment, marked as answer]
This is the default behavior of [Zero Config](https://zeit.co/blog/zero-config). You have a few choices:

- Move "package.json" inside the `api/` folder.
- Do a "no-op" in your build command: `mkdir public`.
- Use a single `index.html` and set a build command like `mkdir public && mv index.html public/`

Let us know how it goes.
